exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb

```diff
@@ -36,20 +36,22 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #246.                header_lbl.value = ITDEStatus.disconnected.value
</pre><p>Shouldn't be?</p>

<pre class="ql-syntax" spellcheck="false">header_lbl.value = ITDEStatus.running.value
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>The names of the Enums are a bit confusing, I agree. The meaning of this state is that the Docker Container is running, but not connected to the network. In the notebook-connector (a helper module, where much of the Python code of the AI-Lab is moved) there are two flags - RUNNING and VISIBLE. In this state their values are respectively True and False. In the front-end this state is called DISCONNECTED because that's how it is displayed. Not ideal, but I can't think of a good fix.</p>

Reply:
<p>Understood.</p>